hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -0,0 +1,47 @@
+{% extends "event/base.html" %}
+
+{% block body %}
+<div class="container">
+    <div class="section">
+        <div class="borderTopDiv z-depth-3">
+            <h1 class="formH1 col s12">Dashboard</h1>
+
+            <h4 class="formH2 col s12">Complete your application</h4>
+            <p><strong>Status</strong>: incompleted</p>
+            <a class="btn-small waves-effect waves-light secondaryColor colorBtn center">Get started</a>
+        </div>
+
+        <div class="borderTopDiv z-depth-3">
+            <h4 class="formH2 col s12">Apply as team</h4>
+
+            <p>Have friends you want to work with? Create a team with up to 4 people and we’ll review your applications together. Enter your team code or make a new one and share with your members.</p>
```

Comment:
From an implementation perspective this year, everyone will be on a team by default. They can join another team and automatically leave theirs, and when they leave another team they'll be automatically put in a new team of 1.

Reply:
So will they have a code displayed already? I'll fix this dashboard page in a future PR. Just wanted the shell already done

Reply:
Yes, they'll already have a team code